<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" name="tests.InvalidsignupPage" tests="2" failures="0" timestamp="09 Nov 2017 06:39:33 GMT" time="30.264" errors="1">
  <testcase name="preConditions" time="30.264" classname="tests.InvalidsignupPage">
    <error type="org.openqa.selenium.WebDriverException" message="Process unexpectedly closed with status: 0
Build info: version: &#039;3.5.3&#039;, revision: &#039;a88d25fe6b&#039;, time: &#039;2017-08-29T12:54:15.039Z&#039;
System info: host: &#039;Unknown&#039;, ip: &#039;Unknown&#039;, os.name: &#039;Linux&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;4.10.0-38-generic&#039;, java.version: &#039;1.8.0_131&#039;
Driver info: driver.version: FirefoxDriver
remote stacktrace: stack backtrace:
   0:  0x812dd47 - backtrace::backtrace::trace::h736111741fa0878e
   1:  0x812dd83 - backtrace::capture::Backtrace::new::h63b8a5c0787510c9
   2:  0x8087efb - webdriver::error::WebDriverError::new::hea6d4dbf778b2b24
   3:  0x808df9b - geckodriver::marionette::MarionetteHandler::create_connection::hf8b6061dba65cdd0
   4:  0x8070e74 - &lt;webdriver::server::Dispatcher&lt;T, U&gt;&gt;::run::hba9181b5aacf8f04
   5:  0x804a756 - std::sys_common::backtrace::__rust_begin_short_backtrace::h19de262639927233
   6:  0x80529f7 - std::panicking::try::do_call::h6c1659fc4d01af51
   7:  0x821cf42 - panic_unwind::__rust_maybe_catch_panic
                at /checkout/src/libpanic_unwind/lib.rs:98
   8:  0x806769b - &lt;F as alloc::boxed::FnBox&lt;A&gt;&gt;::call_box::h953e5f59694972c5
   9:  0x82146bd - alloc::boxed::{{impl}}::call_once&lt;(),()&gt;
                at /checkout/src/liballoc/boxed.rs:661
                 - std::sys_common::thread::start_thread
                at /checkout/src/libstd/sys_common/thread.rs:21
                 - std::sys::imp::thread::{{impl}}::new::thread_start
                at /checkout/src/libstd/sys/unix/thread.rs:84">
      <![CDATA[org.openqa.selenium.WebDriverException: Process unexpectedly closed with status: 0
Build info: version: '3.5.3', revision: 'a88d25fe6b', time: '2017-08-29T12:54:15.039Z'
System info: host: 'Unknown', ip: 'Unknown', os.name: 'Linux', os.arch: 'amd64', os.version: '4.10.0-38-generic', java.version: '1.8.0_131'
Driver info: driver.version: FirefoxDriver
remote stacktrace: stack backtrace:
   0:  0x812dd47 - backtrace::backtrace::trace::h736111741fa0878e
   1:  0x812dd83 - backtrace::capture::Backtrace::new::h63b8a5c0787510c9
   2:  0x8087efb - webdriver::error::WebDriverError::new::hea6d4dbf778b2b24
   3:  0x808df9b - geckodriver::marionette::MarionetteHandler::create_connection::hf8b6061dba65cdd0
   4:  0x8070e74 - <webdriver::server::Dispatcher<T, U>>::run::hba9181b5aacf8f04
   5:  0x804a756 - std::sys_common::backtrace::__rust_begin_short_backtrace::h19de262639927233
   6:  0x80529f7 - std::panicking::try::do_call::h6c1659fc4d01af51
   7:  0x821cf42 - panic_unwind::__rust_maybe_catch_panic
                at /checkout/src/libpanic_unwind/lib.rs:98
   8:  0x806769b - <F as alloc::boxed::FnBox<A>>::call_box::h953e5f59694972c5
   9:  0x82146bd - alloc::boxed::{{impl}}::call_once<(),()>
                at /checkout/src/liballoc/boxed.rs:661
                 - std::sys_common::thread::start_thread
                at /checkout/src/libstd/sys_common/thread.rs:21
                 - std::sys::imp::thread::{{impl}}::new::thread_start
                at /checkout/src/libstd/sys/unix/thread.rs:84
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$new$0(W3CHandshakeResponse.java:57)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$getResponseFunction$2(W3CHandshakeResponse.java:104)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$24(ProtocolHandshake.java:359)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126)
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:362)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:136)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:82)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:646)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:255)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:237)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:138)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:191)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:103)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:99)
	at generics.BaseTest.preConditions(BaseTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:599)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- preConditions -->
  <testcase name="testInvalidsignuppage" time="0.000" classname="tests.InvalidsignupPage">
    <skipped/>
  </testcase> <!-- testInvalidsignuppage -->
</testsuite> <!-- tests.InvalidsignupPage -->
